README.md

```diff
@@ -53,7 +53,7 @@ $ rake dev:setup
 $ heroku local -f Procfile.dev
 # if you chose the local route, then you are good to go on:
   http://localhost:5000
-  $ rspec (to run the test suite)
+  $ rspec (to run the test suite) or bundle exec rspec (if the first does not work)
```

Comment:
p.s. best practice is to make a separate PR for a change like this so the only code in any PR is related to the issue/topic at hand, but it wasn't worth it to hold up the PR any longer this time